scanner/rails.go

```diff
@@ -78,6 +117,38 @@ func configureRails(sourceDir string) (*SourceInfo, error) {
 		}
 	}
 
+	rake := strings.TrimSpace(`
+# commands used to deploy a Rails application
+namespace :fly do
+
+  # BUILD step:
+  #  - changes to the fs make here DO get deployed
+  #  - NO access to secrets, volumes, databases
+  #  - Failures here prevent deployment
+  task :build => 'assets:precompile'
+
+  # RELEASE step:
+  #  - changes to the fs make here are DISCARDED
+  #  - access secrets, databases
+  #  - Failures here prevent deployment
+  task :release => 'db:migrate'
+
+  # SERVER step:
+  #  - changes to the fs make here are deployed
+  #  - access secrets, databases
+  #  - Failures here result in VM being stated, shutdown, and rollback
+  #    to last successful VM.
+  task :server do
+    sh 'bin/rails server'
+  end
+end
+`)
+
+	_, err = os.Stat("lib/tasks/fly.rake")
+	if errors.Is(err, os.ErrNotExist) {
+		os.WriteFile("lib/tasks/fly.rake", []byte(rake), 0o600)
```

Comment:
This could be done by adding this file in the same location as the Dockerfile, so it's installed by `template`, rather than writing out this way. Is there a reason you didn't do it that way?

Reply:
Good catch!  Fixed.